src/test/java/AlloTest/AssertElementsOnPages/AgreementTestRuText.java

```diff
@@ -0,0 +1,40 @@
+package AlloTest.AssertElementsOnPages;
+
+import AlloTest.Pages.TextLocators;
+import ClasesToAllUs.TestInit;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class AgreementTestRuText extends TestInit {
+
+    TextLocators locators;
+    String word;
+    boolean myBooleon;
+
+    @Test
+    public void  agreementTestRuText(){
+        locators = new TextLocators (driver);
+        openUrl("https://allo.ua/ru/");
+        getCheckAssert();
+        resultCheckAseert();
+        System.out.println (myBooleon);
+        Assert.assertTrue (myBooleon);
+    }
+    private void resultCheckAseert() {
+        String myRuWord = "Условия пользовательского соглашения";
+        if (myRuWord.equals(word)){
```

Comment:
out can do this check in one line 
```suggestion
        return myRuWord.equals(word);
```